test/unit/assorted/connection_monitoring_and_pooling.spec.test.ts

```diff
@@ -0,0 +1,37 @@
+import { isHello } from '../../../src/utils';
+import { loadSpecTests } from '../../spec';
+import { cmapTest, runCmapTest, ThreadContext } from '../../tools/cmap_spec_runner';
+import * as mock from '../../tools/mongodb-mock/index';
+
+describe('Connection Monitoring and Pooling Spec Tests', function () {
+  let hostAddress, threadContext;
+  after(() => mock.cleanup());
+  before(async () => {
+    const server = await mock.createServer();
```

Comment:
Not something to do here but do we actually have a ticket to move off this or do we all just share the sentiment? I feel like for simple rounttripping of hellos there's something with sinon that we could do. But we can defer.

Reply:
out of scope, I don't know about a ticket, there's higher priority things to do - it works for now